docs/import/create_score_asset.js

```diff
@@ -411,14 +410,12 @@ function calculateDamage(assetData, setMapBoundsInfo) {
  *
  * This method will go away or be greatly changed if we're using CrowdAI data
  * instead of previously computed building data.
- * @param {ee.Geometry.Polygon} damageEnvelope Area we are concerned with
  * @param {string} buildingPath location of buildings asset in EE
  * @param {ee.FeatureCollection} stateGroups Collection with block groups
  * @return {ee.Dictionary} Number of buildings per block group
  */
-function computeBuildingsHisto(damageEnvelope, buildingPath, stateGroups) {
-  const buildings =
-      ee.FeatureCollection(buildingPath).filterBounds(damageEnvelope);
+function computeBuildingsHisto(buildingPath, stateGroups) {
+  const buildings = ee.FeatureCollection(buildingPath);
```

Comment:
So now it's all the buildings in the state? Have you run it/seen how it performs?

Reply:
It's not really, because we're doing the Join immediately below, which is basically what EE recommended we do for a geometry filter anyway. I haven't run it yet, but I don't see another correct option. Do you?

Reply:
We could try a much bigger envelope than 1 km? Like try to get a sense of biggest block group in the US and go of that, even if it was way bigger than we needed still better than the whole state right. But good with this for now. Assuming this won't be so slow it kill EE right? We used to do this right?